src/views/Settings.vue

```diff
@@ -88,14 +88,14 @@ export default defineComponent({
   methods: {
     setEComStore(store: any) {
       if(this.userProfile) {
-        this.store.dispatch('user/setEcomStore', {
-          'eComStore': this.userProfile.stores.find((str: any) => str.productStoreId == store['detail'].value)
-        })
+        const productStoreId = store['detail'].value
+        this.store.dispatch('user/setEcomStore', { 'productStoreId': productStoreId })
```

Comment:
Why not?
```suggestion
        this.store.dispatch('user/setEcomStore', { 'productStoreId': store['detail'].value })
```

Reply:
Also, we could rename store to event like `setShopifyConfig()` method and access directly event.detail.value